tests/assets/nidcpower_measurement.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package ni.measurementlink.measurement.tests;
+
+message NIDCPowerConfigurations {
+    repeated string pin_names = 1;
+    bool multi_session = 2;
+}
+
+message NIDCPowerOutputs {
+    repeated string session_names = 1;
+    repeated string resource_names = 2;
+    repeated string channel_lists = 3;
+    repeated string channels_connected = 4;
```

Comment:
Rename to `connected_channels` for consistency and because it reads better.

Reply:
My bad, I missed updating the name in some of it's occurrences. Thanks, I've updated them now.